pkg/R/sparkRClient.R

```diff
@@ -11,10 +11,40 @@ connectBackend <- function(hostname, port, timeout = 60) {
   con <- socketConnection(host = hostname, port = port, server = FALSE,
                           blocking = TRUE, open = "wb", timeout = timeout)
 
+  # Set TCP NODELAY for sockets on Unix-like machines.
+  if (.Platform$OS.type == "unix") {
+    # Get the FD for this socket connection
+    sockfd <- getFD(hostname, port)
+    if (sockfd > 0) {
+      ret <- .Call("setTcpNoDelay", sockfd)
+      if (ret == 1) {
+        cat("Successfully set TCP NODELAY on ", sockfd, "\n")
```

Comment:
"Successfully set TCP NODELAY on  4 " ([Travis log](https://travis-ci.org/amplab-extras/SparkR-pkg/builds/48546586)) -- `sockfd` getting printed as a number is for debugging only? When I saw this message I initially thought it's "on 4" or something...